assets/js/app.js

```diff
@@ -161,48 +161,4 @@ photoUpload.onchange = function () {
 	    reader.readAsDataURL(event.target.files[i]);
     }
   }
-};
-
-filterSelection("all")
-function filterSelection(c) {
-  var x, i;
-  x = document.getElementsByClassName("filterDiv");
-  if (c == "all") c = "";
-  for (i = 0; i < x.length; i++) {
-    w3RemoveClass(x[i], "show");
-    if (x[i].className.indexOf(c) > -1) w3AddClass(x[i], "show");
-  }
-}
-
-function w3AddClass(element, name) {
-  var i, arr1, arr2;
-  arr1 = element.className.split(" ");
-  arr2 = name.split(" ");
-  for (i = 0; i < arr2.length; i++) {
-    if (arr1.indexOf(arr2[i]) == -1) {
-      element.className += " " + arr2[i];
-    }
-  }
-}
-
-function w3RemoveClass(element, name) {
```

Comment:
Why did you instead delete this code?

Reply:
I deleted it because I was dint use that code I was trying out something new but it didn't work out but I forgot to clean my code